status-server/src/index.js

```diff
@@ -10,7 +10,6 @@ var dataDir = path.join(__dirname, '../data');
 var publicDir = path.join(__dirname, 'public');
 var config = {
   port: 3000,
-  prefix: 'status'
```

Comment:
lets keep the 'prefix' config, but use it instead to parameterize the 'api' string we are using in the routes/URLs